packages/api/src/controllers/stripe.ts

```diff
@@ -261,29 +327,42 @@ app.post("/webhook", async (req, res) => {
   } else if (event.type === "invoice.payment_failed") {
     // Notify via email
     const invoice = event.data.object;
+    let invoiceUrl = invoice.invoice_pdf;
```

Comment:
Is this right? A "url" is assigned from a "pdf", might be a mistake here.

Reply:
Yeah, it's a direct link to the invoice so when shih-yu receive the email, he can check it. It's coming with the stripe webhook, and this is the only useful url, unless I build one myself that points to the dashboard